### PR TITLE
pythonPackages.isbnlib: clarify license

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/isbnlib/default.nix [hunk 1]
```diff
@@ -27,7 +27,7 @@ buildPythonPackage rec {
   meta = with lib; {
     description = "Extract, clean, transform, hyphenate and metadata for ISBNs";
     homepage = "https://github.com/xlcnd/isbnlib";
-    license = licenses.lgpl3;
+    license = licenses.lgpl3Only;
```
Comment (fabaff) [inline pkgs/development/python-modules/isbnlib/default.nix:30]: It looks like `LGPLv3+` to me ([COPYRIGHT.txt](https://github.com/xlcnd/isbnlib/blob/dev/COPYRIGHT.txt#L5-L8) and in the file headers, e.g., [here](https://github.com/xlcnd/isbnlib/blob/dev/isbnlib/_core.py#L5-L10)). To be `LGPLv3`, the license statement doesn't have to contain "..., or (at your option) any later version". 

```suggestion
    license = licenses.lgpl3Plus;
```

**Review by mweinelt — APPROVED** [review]

File: pkgs/development/python-modules/isbnlib/default.nix (hunk 1 above)
Comment (mweinelt) [reply pkgs/development/python-modules/isbnlib/default.nix:30]: You are right, I searched for "newer", not "later" and therefore missed it.

Comment (dotlambda) [reply pkgs/development/python-modules/isbnlib/default.nix:30]: Oh true, but this file says gpl3Only: https://github.com/xlcnd/isbnlib/blob/dev/LICENSE-LGPL-3.0-only.txt

Comment (dotlambda) [reply pkgs/development/python-modules/isbnlib/default.nix:30]: I opened https://github.com/xlcnd/isbnlib/issues/77.

Comment (dotlambda) [reply pkgs/development/python-modules/isbnlib/default.nix:30]: Correcting the license in https://github.com/NixOS/nixpkgs/pull/119302.

**Pull request merged**